### ISSUE TITLE
Put a large format poster on the wall beside the outside front door

### ISSUE THREAD

**Issue by ajlennon** [DoESLiverpool/somebody-should#1690]
It occurred to me yesterday that there are lots of students and others wandering outside along Kempston Street.

Given that DoES wishes to increase diversity, partly through inviting younger people into the space would it not make some sense to let passers by know the space is here?

I was thinking maybe an A0 waterproof/vinyl printed poster based on the DoES mission statement collateral inviting people to come in and say hello?  (Subject to Jason's approval of course but it could fit in the space between the door and the existing murals)

**Comment by JackiePease** [comment]
@ajlennon and I spoke to Jason - he's not against the idea, but would like to see what the poster looks like before we go ahead. @ajlennon is going to get something mocked up, perhaps will ask @zarino for input

**Comment by ajlennon** [comment]
Yes would definitely appreciate @zarino 's thoughts as I have no idea about creative stuff :-D
